### ISSUE TITLE
Resuts/Contribution PDF

### ISSUE THREAD

**Issue by pGit1** [erko/nips17-defense#2]
Hello,
Can you please provide the summary of your project and also the accuracy results?
Thank you.

**Comment by erko** [comment]
Hi @pGit1,
Added poster files describing about my solution (were hanged at NIPS 2017 workshop).
Very soon will be posted NIPS competitions track book here on github. There my and other solutions were described more detailed.

Particularly about my solution, in two words: very simple defense tactic using median filtering fed to _only_ adversarially trained models. In my submitted solution I've used ensemble of adversarially pretrained models listed in this repo: https://github.com/tensorflow/models/. So, doesn't even need additional expensive trainings if you already adversarially trained models. 
Final results: 4th place on leaderboard.

This solution are applicable to big images like in ImageNet and not general. So, problem is far from solved.

**Comment by pGit1** [comment]
THANK YOU!

**Comment by pGit1** [comment]
@erko 

Could you provide a link to this: 

> Very soon will be posted NIPS competitions track book here on github

Thank you! Great job by the way

**Comment by erko** [comment]
@pGit1
It up to NIPS organizers, they asked to not share it before their release.. So, I'm waiting them too.
